Modules/Translation/Resources/lang/core/en/core.php

```diff
@@ -77,7 +77,7 @@
     'unauthorized' => 'Unauthorized',
     'unauthorized-access' => 'You do not have the appropriate permissions to access that page.',
     'unauthenticated' => 'Niet ingelogd',
-    'unauthenticated-access' => 'U moet inloggen om toegang te krijgen tot deze pagina',
+    'unauthenticated-access' => 'You need to be logged in to be able to view this page',
```

Comment:
Why was the Dutch translation removed?

Reply:
This is in the english translation file, doesn't seem like it belonged here